opentreemap/treemap/lib/map_feature.py

```diff
@@ -303,8 +309,41 @@ def context_dict_for_map_feature(request, feature):
         'address_full': feature.address_full,
         'upload_photo_endpoint': None,
         'photos': None,
+        'share': None,
     }
 
     _add_eco_benefits_to_context_dict(instance, feature, context)
 
     return context
+
+
+def _add_share_context(context, request, photos):
+    if len(photos) > 0:
+        photo_url = photos[0].thumbnail.url
+    elif context.get('has_tree', False):
+        photo_url = settings.STATIC_URL + "img/tree.png"
+    else:
+        photo_url = settings.STATIC_URL + "img/otmLogo126.png"
+    photo_url = request.build_absolute_uri(photo_url)
+
+    title = trans("%s on OpenTreeMap site %s") % (context['title'],
```

Comment:
"Some title on OpenTreeMap site PhillyTreeMap" is a little wordy compared to "Some title on PhillyTreeMap." Is getting the OTM branding in there a specific part of the feature?